share/dkg/pedersen/dkg.go

```diff
@@ -242,6 +247,14 @@ func (d *DistKeyGenerator) ProcessJustification(j *Justification) error {
 	return v.ProcessJustification(j.Justification)
 }
 
+// SetTimeout triggers the timeout on all verifiers, and thus makes sure
+// all verifiers have either responded, or have a StatusComplaint response.
```

Comment:
The user should only need to care about "what this functions does, when should I call it, what's the output"
so more something like:
```
SetTimeout marks the end of a round, invalidating any missing (or future) response for this DKG protocol round. The caller is expected to call this after a long timeout so each DKG node can still compute its share if enough Deals are valid.
```
stg along those lines.